rollup.config.js

```diff
@@ -1,11 +1,29 @@
 import babel from 'rollup-plugin-babel';
+import typescript from 'rollup-plugin-typescript2';
+
 import pkg from './package.json';
 
 export default [
   {
-    input: 'src/index.js',
-    output: [{ file: pkg.main, format: 'esm' }],
+    input: 'src/index.ts',
+    output: [
+      {
+        file: pkg.main,
+        format: 'cjs'
+      },
+      {
+        file: pkg.module,
+        format: 'es'
```

Comment:
Tiny nit: the latest rollup.js docs on output format suggests `esm`:

![image](https://user-images.githubusercontent.com/45576380/71284318-78c80c00-2330-11ea-82d3-8515b5ca34a1.png)

https://rollupjs.org/guide/en/